README.md

```diff
@@ -11,6 +11,10 @@
 
 By using the kubernetes [vertical-pod-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) in recommendation mode, we can see a suggestion for resource requests on each of our apps. This tool creates a VPA for each deployment in a namespace and then queries them for information.
 
+<div align="center">
```

Comment:
Can you put some text above this to explain what we are seeing here?  I don't think it's clear that this is the Goldilocks dashboard.  Perhaps it should be placed in a different spot?

Reply:
Good point!